gpii/node_modules/installOnDemand/src/installOnDemand.js

```diff
@@ -0,0 +1,663 @@
+/*
+ * Install on Demand.
+ *
+ * Copyright 2018 Raising the Floor - International
+ *
+ * Licensed under the New BSD license. You may not use this file except in
+ * compliance with this License.
+ *
+ * The R&D leading to these results received funding from the
+ * Department of Education - Grant H421A150005 (GPII-APCP). However,
+ * these results do not necessarily represent the policy of the
+ * Department of Education, and you should not assume endorsement by the
+ * Federal Government.
+ *
+ * You may obtain a copy of the License at
+ * https://github.com/GPII/universal/blob/master/LICENSE.txt
+ */
+
+"use strict";
+
+var fluid = require("infusion");
+
+var path = require("path"),
+    os = require("os"),
+    fs = require("fs"),
+    request = require("request"),
+    glob = require("glob");
+
+var gpii = fluid.registerNamespace("gpii");
+fluid.registerNamespace("gpii.iod");
+
+require("./packageInstaller.js");
+require("./iodSettingsHandler.js");
+
+/**
+ * Information about a package.
```

Comment:
Link to the README and/or IoD Technical design document would be helpful in interpreting this